### DISCUSSION TITLE
Will the microphone work after rooting

### DISCUSSION THREAD

**Discussion by roschi02** [ThomasPrior/LenovoSmartClock2#12] [Q&A]
Hi, I have a basic question: will the microphones work after rooting the device?
I'm asking because I followed the steps described in this [video](https://youtu.be/t0hp2xTHsqk?si=SHiHLB72SKaGBZKh), but I can't get the microphones to work, either with Home Assistant (my intended use case) or even with a recorder for testing. I can only hear something if I bang the device on the table, but it's very quiet.

Another possibility is that my device might be broken.
Do you have any ideas about what could be causing this?

**Comment by ThomasPrior** [comment]
Nothing about this rooting process targets the microphones specifically and some of the people who have purchased rooted units from me have gone on to use them as VoIP endpoints. Before disabling the Google launcher you can still invoke assistant with "OK Google" so the microphone definitely works.

 I'd argue either the device has the microphone button turned off, is broken or there's some kind of permissions issue.
